Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -405,64 +405,64 @@ Enabled filters have priority from top to bottom in your list. Filter precedence
     <string name="setting_layout_mode">Layout mode</string>
 
     <!-- Appearance layout group -->
-    <string name="setting_board_grid_span_count_portrait">Catalog mode column count (portrait)</string>
-    <string name="setting_album_grid_span_count_portrait">Album mode column count (portrait)</string>
-    <string name="setting_board_grid_span_count_landscape">Catalog mode column count (landscape)</string>
-    <string name="setting_album_grid_span_count_landscape">Album mode column count (landscape)</string>
+    <string name="setting_board_grid_span_count_portrait">Catalog mode in columns (portrait)</string>
```

Comment:
Not sure about these

Reply:
Yes I would leave this alone. The string itself refers to how many columns you want to appear in the relevant mode(s).
Looking on how this string is used in settings, your suggestion doesn't work.

Reply:
```suggestion
    <string name="setting_board_grid_span_count_portrait">Columns in catalog mode (portrait)</string>
```